index.html

```diff
@@ -52,5 +53,124 @@ <h1>
         <a href=""><img src="images/Disabled.svg" alt="" /></a>
       </div>
     </div>
+
+    <!-- The header for my recent works -->
+
+    <div class="top-tile">
+      <h1 class="card-title">
+        My Recent <br />
+        Works
+      </h1>
+      <hr />
+    </div>
+    <!-- The main card container -->
+    <div class="card-container">
+      <div class="card">
+        <div class="bottom-card">
+          <h4>Multi-Post Stories <br />Gain+Glory</h4>
+          <ul>
+            <div class="top-card"></div>
+            <li>Html</li>
+            <li>Css</li>
+            <li>Javascript</li>
+          </ul>
+          <button class="card-button">See Projects</button>
+        </div>
+      </div>
+
+      <div class="card">
+        <div class="bottom-card">
+          <h4>Multi-Post Stories <br />Gain+Glory</h4>
+          <ul>
+            <div class="top-card"></div>
+            <li>Html</li>
+            <li>Css</li>
+            <li>Javascript</li>
+          </ul>
+          <button class="card-button">See Projects</button>
+        </div>
+      </div>
```

Comment:
- [ ] Kindly add a margin between the language skills in your cards to match the original design

![image](https://user-images.githubusercontent.com/98453979/205034504-1c10861b-496e-4004-a271-58056dda67a0.png)

### Checkout Figma

![image](https://user-images.githubusercontent.com/98453979/205034670-fec78f75-7566-42ea-8408-a14f23cda931.png)

- [ ] Also kindly adjust the border-color of your buttons to match the Figma design

![image](https://user-images.githubusercontent.com/98453979/205035057-3229cda7-046a-43d0-8557-a0fe6a55e37f.png)

### Checkout Figma

![image](https://user-images.githubusercontent.com/98453979/205035305-cd4c24f6-9973-46c0-bf86-fe9d41394e7b.png)

Reply:
Thank you for your review. As regards the border-color, I copied the style directly from figma. But I will adjust it